specification/cognitiveservices/OpenAI.Inference/models/completions/azure_content_filtering.tsp

```diff
@@ -138,6 +138,11 @@ model ContentFilterResultsForChoice {
   @encodedName("application/json", "protected_material_code")
   @doc("Information about detection of protected code material.")
   protectedMaterialCode?: ContentFilterCitedDetectionResult;
+
+  @added(ServiceApiVersions.v2024_10_01_Preview)
+  @encodedName("application/json", "ungrounded_material")
+  @doc("Information about detection of ungrounded material.")
+  ungroundedMaterial: ContentFilterCompletionTextSpanResult;
```

Comment:
This should be optional, no?
```suggestion
  ungroundedMaterial?: ContentFilterCompletionTextSpanResult;
```

Reply:
I am looking at https://github.com/Azure/azure-rest-api-specs/blob/0035f4bd79ef31f8d4e3303ed0a38c1087f57fcf/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-10-01-preview/inference.json#L3276 and I don't see `ungrounded_material` referenced in a `required` list.